specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-12-01-preview/inference.json

```diff
@@ -0,0 +1,12335 @@
+            "description": "Sort order by the `created_at` timestamp of the objects. `asc` for ascending order and `desc` for descending order.\n",
+            "schema": {
+              "type": "string",
+              "default": "desc",
+              "enum": [
+                "asc",
+                "desc"
+              ],
+              "x-ms-enum": {
+                "name": "ListRunsOrder",
+                "modelAsString": true,
+                "values": [
+                  {
+                    "value": "asc",
+                    "description": "Order results in ascending order"
+                  },
+                  {
+                    "value": "desc",
+                    "description": "Order results in descending order"
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "name": "after",
+            "in": "query",
+            "description": "A cursor for use in pagination. `after` is an object ID that defines your place in the list. For instance, if you make a list request and receive 100 objects, ending with obj_foo, your subsequent call can include after=obj_foo in order to fetch the next page of the list.\n",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "before",
+            "in": "query",
+            "description": "A cursor for use in pagination. `before` is an object ID that defines your place in the list. For instance, if you make a list request and receive 100 objects, starting with obj_foo, your subsequent call can include before=obj_foo in order to fetch the previous page of the list.\n",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "example": "2024-12-01-preview",
+              "description": "api version"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/listRunsResponse"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all runs.": {
+            "$ref": "./examples/list_runs.json"
+          }
+        }
+      },
+      "post": {
+        "operationId": "Create_Run",
+        "tags": [
+          "Assistants"
+        ],
+        "summary": "Create a run.",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "thread_id",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The ID of the thread to run."
+          },
+          {
+            "name": "include[]",
```

Comment:
This name looks wrong?

Reply:
As discussed offline, this is aligned with the OAI spec.